luxonis_ml/data/augmentations/augmentations.py

```diff
@@ -85,41 +77,34 @@ def from_config(
             )
             if isinstance(curr_aug, A.ImageOnlyTransform):
                 pixel_augs.append(curr_aug)
-            elif isinstance(curr_aug, A.DualTransform):
-                spatial_augs.append(curr_aug)
             elif isinstance(curr_aug, BatchBasedTransform):
                 batch_size *= curr_aug.batch_size
                 batched_augs.append(curr_aug)
+            elif isinstance(curr_aug, A.DualTransform):
+                spatial_augs.append(curr_aug)
 
-        def _get_params(batch: bool = False):
-            suffix = "_batch" if batch else ""
+        def _get_params():
             return {
                 "bbox_params": A.BboxParams(
-                    format="coco",
-                    label_fields=[
-                        f"bboxes_classes{suffix}",
-                        f"bboxes_visibility{suffix}",
-                    ],
+                    format="albumentations",
+                    label_fields=["bboxes_classes", "bboxes_visibility"],
+                    # Bug in albumentations v1.4.18 (the latest installable
+                    # on python 3.8) causes the pipeline to eventually
+                    # crash when set to 0.
```

Comment:
When what is set to 0? Do we want to exclude this version from requrements if we know it causes problems?

Reply:
It's 0 by default (bbox must be completely outside of the image to be excluded), but for some reason this is buggy up until and including `1.4.18`. Unfortunately, this is the latest version available for Python 3.8.

Reply:
I'd argue this is virtually correct. But we can make the number even smaller.

Reply:
Yeah if the issue is only the exact 0 and some float close to 0 doesn't cause it then I think it's fine to set it like that. And make a comment why this is like that so we remember not to change it